src/coreclr/vm/interpexec.h

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef _INTERPEXEC_H_
+#define _INTERPEXEC_H_
+
+#include "../interpreter/interpretershared.h"
+
+#define INTERP_STACK_SIZE 1024*1024
+
+struct StackVal
+{
+    union
+    {
+        int32_t i;
+        int64_t l;
+        float f;
+        double d;
+        void *o;
+        void *p;
+    } data;
+};
+
+struct InterpFrame
```

Comment:
We will need to also have an explicit frame derived from the `Frame` class (in frames.h) created at the entry point to the interpreter when JITed / crossgened code calls interpreted method for the purpose of stack walking. I will add that one in my follow up change, but since those frames are all named "XXXFrame", it would be confusing to have the `InterpFrame` here named similarly. That was also one of the feedbacks that @kg has provided to one of the documents I've produced during the interpreter analysis experiment.
So I'd like this to be named somehow differently. My plan was to name the other one `InterpreterFrame`. What would you think about renaming this one to `InterpContext`?